common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java

```diff
@@ -262,6 +262,28 @@ public void loadAvatar(@LuaNotNil String playerUUID) {
         NetworkStuff.getUser(new UserData(uuid));
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc(
+        overloads = {
+            @LuaMethodOverload(
+                argumentTypes = { String.class },
+                argumentNames = { "playerUUID" }
+            )
+        },
+        value = "goofy.get_player_color"
```

Comment:
now update this to match
```suggestion
        value = "goofy.get_avatar_color"
```